crates/storage_impl/src/lookup.rs

```diff
@@ -50,7 +50,7 @@ impl<T: DatabaseStore> ReverseLookupInterface for RouterStore<T> {
                     er.change_context(new_err)
                 })
         };
-        get_or_populate_redis(self, id, database_call).await
+        get_or_populate_redis(self, format!("reverse_lookup_{id}"), database_call).await
```

Comment:
In all reverse lookup are we adding prefix as `reverse_lookup_`?

Reply:
Yeah because this doesn't have shard key so it needs a prefix to avoid key clashing.